_rules/scrollable-element-keyboard-accessible-0ssw9k.md

```diff
@@ -46,6 +44,9 @@ _No accessibility support issues known._
 
 ## Background
 
+To ensure there is some element from which arrow keys can be used to control the scroll position, focus must be on or in a scrollable region. 
+Tf scripts are used to prevent the keyboard events from reaching the scrollable region, this could still cause a keyboard accessibility issue. This must be tested separately.
```

Comment:
```suggestion
To ensure there is some element from which arrow keys can be used to control the scroll position, focus must be on or in a scrollable region. If scripts are used to prevent the keyboard events from reaching the scrollable region, this could still cause a keyboard accessibility issue. This must be tested separately.
```